appveyor.yml

```diff
@@ -0,0 +1,20 @@
+version: 1.0.{build}-{branch}
```

Comment:
Can we make this change separately? Adding AppVeyor should be in its own PR/commit.

Reply:
I wanted to ensure that appveyor was running successfully on this pr, but this can be implemented on another pr.